src/app/options/options.component.html

```diff
@@ -0,0 +1,6 @@
+<div >
+  <h1><font size="5">Options:</font></h1>
+  <ul>
+<li><a routerLink="/certificate2"><font size="4">Style1</font></a></li>
+<li><a routerLink="/certificate"><font size="4">Style2</font></a><li></ul>
+  </div>

```

Comment:
this option to select from templates would be provided on organization's dashboard. please remove it from here

Reply:
So I just have to delete the option component  and put the designs on the same page ?